src/App.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+import TodoPage from './TodoPage';
+
+function App() {
+  const [state, setState] = useState({
+    todo: {
+      id: 1,
+      content: '',
+    },
+    todos: [],
```

Comment:
```suggestion
    todo: [{
      id: 1,
      content: '',
    }],
    todos: [],
```
todo가 이런 모양이면 어떤가요? todos와 todo의 역할이 각각 뭔지 이해가 잘 되지 않는 것 같아요~

Reply:
todo의 역할을 명확히 표현 하기 위해서 변수명을 newTodo로 변경하였습니다.

todo가 배열이 되는거 같은데.. 피드백 주신게 잘 이해되지 않습니다 ㅠ.ㅠ